docs/guide/driver-queue-interop.md

```diff
@@ -0,0 +1,51 @@
+Queue Interop Driver (php-enqueue)
+==================================
+
+The driver works with many queue brokers.
+
+Full list of supported brokers you can find on the [Queue Interop](https://github.com/queue-interop/queue-interop) page
+
+To get it works you have to install and setup one of the supported implementation.
+
+Configuration example for the RabbitMQ AMQP:
```

Comment:
I'm not sure it's a good example since there's RabbitMQ driver that works w/o Queue Interop already. It all depends on the goal.

@zhuravljov Do we want to remove our own implementations and use Queue Interop ones instead? That would be less maintenance and more profit for PHP overall. Or do we want to have additional drivers through Queue Interop?